packages/miniflare/package.json

```diff
@@ -30,8 +30,7 @@
 		"bootstrap.js"
 	],
 	"scripts": {
-		"build": "node scripts/build.mjs && pnpm run types:build",
-		"capnp:workerd": "capnpc -o ts src/runtime/config/workerd.capnp",
+		"build": "capnp-es node_modules/workerd/workerd.capnp -ots && mv node_modules/workerd/workerd.ts src/runtime/config/generated.ts && node scripts/build.mjs && pnpm run types:build",
```

Comment:
Is `mv` going to work on Windows?

Generally it is better to put this stuff into a Node.js script I think

Reply:
https://github.com/cloudflare/workers-sdk/pull/7788/commits/c1e6afe138913e5785fa25e1497ea3d0b7db6583